README.md

```diff
@@ -38,20 +38,78 @@ npm run dev
 ```
 
 ## How to i18n
+### Scripts
+We use i18next + gettext convertion approach in our development toolchain.
+There are several scripts in package.json:
 
-We use 18next + gettext convertion approach in our development toolchain.
-There are several tasks in package.json:
 - i18n:export converts i18next to gettext
 - i18n:import converts gettext to i18next
-- i18n:update extracts new translation keys from code to common.json (i18n.t, <Trans/>) and removes unused keys.
-- lint:i18n:keys:identity is needed to see untranslated keys for all locales (make sure you ran i18n:import before, it compares i18n files)
+- i18n:update extracts new translation keys from code to common.json (```i18n.t``` or ```<Trans/>```) and also removes unused keys
+- lint:i18n:keys:identity is needed to see untranslated keys for all locales (make sure you ran i18n:import before as it compares i18next .json files)
 - i18n:msgmerge is needed to keep all .po files synced with .pot file (marks non-compliant msgid as 'fuzzy', renames keys in .po according keys changes in .pot)
```

Comment:
very unobvious command name btw, but as long as we have doc - it's fine
(i'd expect smth like "updatepot" or 'syncpot' or in js world i'd go with "updateTranslationsArchive" but that's too long for those who type in console :D)

Reply:
hm, msgmerge is a name of gettext package util, so that's why  🤷‍♂️ 
maybe i18n:gettext:sync ?